first_milestone/seed.sql

```diff
@@ -0,0 +1,118 @@
+-- In this file we define the schema for the database and create the tables.
+-- You can run the file with the following command:
+--
+--     mysql -u root -p atlargetest < seed.sql
+
+-- First we make sure our database exists and drop any existing tables.
+DROP TABLE IF EXISTS Team, Region, TeamInRegion, AppUser, Admin, Meet, Participates;
+DROP VIEW IF EXISTS TeamWithRegion;
+
+-- AppUser's are useful in web applications for handling API permissions, and 
+-- in the future we may want to add the ability to associate certain data
+-- with the AppUser. For now it is just for the purposes of loggin in / out
+CREATE TABLE AppUser (
+  email varchar(255) NOT NULL,
+  name varchar(255) NOT NULL,
+  hash varchar(255) NOT NULL,
+  PRIMARY KEY (email)
+);
+-- Description of Team
+CREATE TABLE Team (
+  ID int NOT NULL AUTO_INCREMENT,
+  name varchar(255) NOT NULL,
+  gender ENUM('mens', 'womens') NOT NULL,
+  PRIMARY KEY (ID)
+);
+-- Description of TeamInRegion
+CREATE TABLE TeamInRegion (
+  team_id int NOT NULL REFERENCES Team(id),
+  region_id int NOT NULL REFERENCES Region(id),
+  team_rank int NOT NULL,
+  PRIMARY KEY (team_id, region_id)
+);
+-- Description of Region
+CREATE TABLE Region (
+  ID int NOT NULL AUTO_INCREMENT,
+  name varchar(255),
+  PRIMARY KEY (ID)
+);
+-- Description of Meet
+CREATE TABLE Meet (
+  ID int NOT NULL AUTO_INCREMENT,
+  name varchar(255),
+  date date,
```

Comment:
i wonder if naming this "date" will cause problems later/if its treated as a type and thus protected word

Reply:
doesn't in mysql, but I see your point